CONTRIBUTING.md

```diff
@@ -1,10 +1,12 @@
 <!-- omit in toc -->
```

Comment:
?

Reply:
toc was probably autogenerated so that's there, should I remove them?

Reply:
yes pls!